scale/source/configuration/source_data_file.py

```diff
@@ -36,3 +40,38 @@ def save_parse_results(self, parse_results, input_file_ids):
 
             SourceFile.objects.save_parse_results(src_file_id, geo_json, data_started, data_ended, data_types,
                                                   new_workspace_path)
+
+    def save_parse_results_v6(self, id_to_metadata):
+        """Saves the given parse results to the source file for the given ID. All database changes occur in an atomic
+        transaction.
+
+        :param id_to_metadata: Mapping of IDs to metadata objects/
+        :type id_to_metadata: { int: class:`job.seed.metadata.SeedMetadata` }
+        """
+
+        ids = id_to_metadata.keys()
+        logger.debug('List of IDs to update: {}'.format(ids))
```

Comment:
Do we need these debug statements in still?

Reply:
I left them since they were debug only. They were useful to detect which file ids were matched for updating with metadata.